lib/client.js

```diff
@@ -215,6 +215,26 @@ class HttpTransportClient {
     return this;
   }
 
+  /**
+   * Set the redirect handling:
+   *   `follow` (default) to follow the redirects automatically,
+   *   `manual` to extract redirect headers,
+   *   `error` to reject redirect
+   *
+   * @param {'follow'|'manual'|'error'} redirect - redirect handling
+   * @return a HttpTransport instance
+   * @example
+   * const httpTransport = require('@bbc/http-transport');
+   *
+   * const response = await httpTransport.createClient()
+   *    .redirect('manual') // for this request only
+   *    .asResponse();
+   */
+  redirect(redirectType) {
+    this._ctx.redirect = redirectType;
+    return this;
+  }
+
```

Comment:
I think this needs to documented in the similar way to the the `.headers()`, `.query()`, `.retry()` and `timeout()` has been done in `docs.md, `client.js.html` and `index.html`

Reply:
Also we should create a matching 'getter' method, similar to this `getTimeout` method: https://github.com/bbc/http-transport/blob/main/lib/request.js#L57